src/components/login/actions.js

```diff
@@ -0,0 +1,41 @@
+import { LOGIN_USER, AUTHENTICATE_USER } from './actionTypes';
+
+/**
+ * Triggers request to authenticate user and log them in using firebase
+ *
+ * @function
+ * @return {Object} The {@link actionTypes.LOGIN_USER LOGIN_USER} action.
+ */
+const loginUser = password => {
+    // convert the password to string for easy slicing and nomenclature
+    const stringPassword = String(password);
+    const stringPasswordWithNumber = `+234${stringPassword}`;
+
+    // return the actiontype and the password which has been appended with country code
+    return {
+        payload: stringPasswordWithNumber,
+        type: LOGIN_USER,
+    };
+};
+
+/**
+ * Triggers request to NIN API and confirm that user is indeed a citizen
+ *
+ * @function
+ * @return {Object} The {@link actionTypes.AUTHENTICATE_USER AUTHENTICATE_USER} action.
+ */
+const authenticateUser = payload => {
+    // convert the password to string for easy slicing and nomenclature
+    const stringPassword = String(payload.phoneNumber);
+    const password = `+234${stringPassword}`;
+    const { nin } = payload;
+
```

Comment:
i don't really understand why you are appending country code to password.
line 29 - 31 can  be
```
    const { nin, phoneNumber } = payload;
 const stringPassword = String(payload);
    const password = `+234${stringPassword}`;
```

Reply:
i misnamed it as password, which i have corrected. its content is a phone number. we are using phone number to authenticate not password

Reply:
good